keybot/main.go

```diff
@@ -91,7 +91,7 @@ func kingpinKeybotHandler(args []string) (string, error) {
 		if err != nil {
 			return "", err
 		}
-		return slackbot.NewExecCommand("/bin/launchctl", []string{"start", "keybase.prerelease.promotearelease"}, false, "Promote a specific release to public").Run(emptyArgs)
+		return slackbot.NewExecCommand("/bin/launchctl", []string{"start", "keybase.prerelease.promotearelease"}, false, "Promote a release to public, takes an optional specific release").Run(emptyArgs)
```

Comment:
yes, if a release isn't passed in then it just does the `promote-releases` command.